package.json

```
@@ -71,12 +74,23 @@
     "lit": "3.2.1"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "^18.2.3",
```

I suggest to go to Angular 19 for all angular deps
```suggestion
    "@angular-devkit/build-angular": "18.2.3",
```